db/schema.rb

```diff
@@ -10,109 +10,108 @@
 #
```

Comment:
You should skip `schema.rb` on Rubocop because it's autogenerated.

Reply:
I already have done that.